api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -1863,7 +1864,7 @@ public void saveObs_shouldNotVoidAnObsWithNoChanges() {
 	@Test
 	public void saveObs_shouldCopyTheFormNamespaceAndPathFieldInEditedObs() {
 		executeDataSet(INITIAL_OBS_XML);
-		Obs obs = Context.getObsService().getObs(7);
```

Comment:
Why do we change the obs id from 7 to 6?

Reply:
In the dataset the namespace parameter has been shifted from from obs_id=7 to obs_is=6. So that test still accesses the obs with formnamespace parameter, I did this.